app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -61,7 +61,7 @@ def requested_authn_contexts
   end
 
   def requested_authn_context
-    if AppConfig.env.aal_authn_context_enabled == 'true'
+    if IdentityConfig.store.aal_authn_context_enabled
```

Comment:
How would you feel if we updated this to add `?` to the end of `type: :boolean` attributes?

Reply:
Or maybe to improve searchability maybe we have it error if there is no `?`, ex:


```ruby
config.add(:some_boolean?, type: :boolean) # succeeds
config.add(:some_boolean, type: :boolean) # throws an error, booleans must end in `?`
```

Or maybe we add an alias field (aka reader method) so that we have both the original key + the method name, and can error:

```
config.add(:some_boolean, alias: :some_boolean?, type: :boolean) # succeeds
config.add(:some_boolean, alias: :some_boolean, type: :boolean) # errors, alias for boolean must have ?
```

Reply:
I was torn on this, I messed around a little bit with it. The simple version was something along those lines:

```ruby
    if type == :boolean
      @written_env[:"#{key}?"] = converted_value
    end
# OR
     if options[:alias]
      @written_env[alias] = converted_value
    end
```

I ended up deciding not to because it felt simpler to keep this layer as explicit config where methods are tied directly to the keys that represent them. It runs a bit against the Ruby-ism of using the `?` suffix for boolean methods, but it didn't feel like it was worth the indirection here. My feeling is to push things like this out to interfaces like  `FeatureManagement`.

Reply:
that seems reasonable for now. thx!